### DISCUSSION TITLE
Wavelength dependence of the refraction coefficient

### DISCUSSION THREAD

**Discussion by buddhasystem** [mitsuba-renderer/mitsuba3#1226] [Q&A] [answered]
Dear All,

Is it possible to define a dependency of the refraction coefficient in Mitsuba, on the wave length? I assume I'd have to use the "dielectric" material. Thank you!

**Comment by njroussel** [comment, marked as answer]
Hi @buddhasystem 

I guess you're referring to dispersion. This is theoretically possible and pretty easy to implement. However, we have not done it in Mitsuba 3 and there is no plan to do so currently.

**Comment by buddhasystem** [comment]
@njroussel Thank you! So there is no hope of modeling let's say a prism that produces a rainbow out of a beam of white light?

**Reply by njroussel** [reply]
No :(
